spec/core/middleware/response/parse_iso_dates_spec.rb

```diff
@@ -8,13 +8,7 @@ def fake_response(data)
     response
   end
 
-  let(:parsed){
-    if RUBY_VERSION > "1.9"
-      "2012-02-01 13:14:15 UTC"
-    else
-      "Wed Feb 01 13:14:15 UTC 2012"
-    end
-  }
+  let(:parsed) { "2012-02-01 13:14:15 UTC" }
```

Comment:
This is unrelated to the live specs change, I just noticed this very old and very unnecessary Ruby version check.